docs/cloud/satellites/byoc/byoc.md

```diff
@@ -0,0 +1,70 @@
+# Bring Your Own Cloud
+
+"Bring Your Own Cloud" (BYOC) satellites **Experimental** are a hybrid between [self hosted](../self-hosted.md) and [earthly-managed](../../satellites.md) satellites. These satellites are managed by Earthly; just like our managed offering, but within your infrastructure. This blends the ease-of-use of managed satellites with the security benefits that self hosting can bring.
```

Comment:
```suggestion
"Bring Your Own Cloud" (BYOC) satellites **Experimental** are a hybrid between [Self-Hosted](../self-hosted.md) and [Cloud](../../satellites.md) satellites. These satellites are managed by Earthly; just like our managed offering, but within your infrastructure. This blends the ease-of-use of managed satellites with the security benefits that self hosting can bring.
```

We tend to call these "Cloud" satellites in our pricing page and in other [doc pages](https://docs.earthly.dev/docs/remote-runners).

Reply:
I still think we should be consistent with how we name these products in our docs and pricing page. We haven't referred to Earthly-Managed in the other pages, so it might get confusing. 

I also think the term Earthly-managed could include BYOC because Earthly manages those clouds too.